community/modules/compute/schedmd-slurm-gcp-v6-nodeset-tpu/main.tf

```diff
@@ -38,5 +38,8 @@ locals {
     subnetwork      = var.subnetwork_self_link
     service_account = var.service_account
     zone            = var.zone
+
+    project_id = var.project_id
```

Comment:
You will also need to add those fields to the `community/modules/compute/schedmd-slurm-gcp-v6-partition/variables.tf` & `community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables.tf` `variable "nodeset_tpu"` . Otherwise during type casting they will be lost.

Reply:
I missed that. thanks for pointing out. Updated.